usr/local/sbin/entrypoint.d/05-deskpro-cloud.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#######################################################################
+# This source handles setup steps for Deskpro Cloud.
+#######################################################################
+
+function deskpro_cloud_main() {
+  if [ -z "$DESKPRO_CLOUD_MODE" ]; then
+    boot_log_message TRACE "[cloud] Not running in Cloud mode, skipping cloud setup."
+    return
+  fi
+  cp -r /srv/deskpro/cloud/. /srv/deskpro/INSTANCE_DATA
```

Comment:
What's this do?

Reply:
This copies the cloud config into INSTANCE_DATA. We did have it in the cloud deployment steps but we were looking to make it more uniform and potentially speed up the cloud deployment process. A couple of bits need a bit more thought but this step can be moved here from the docker file

Reply:
Where is it used though? Just copying files into INSTANCE_DATA isn't going to actually do anything

Reply:
We use the envs when running in cloud to point at the cloud config files (like we were doing previously)

Reply:
What envs? If there's an env with a path, why not just set the actual proper path? I feel like `cp -r`ing these files is fragile

Reply:
(Not against this if it's the only clean way, just don't want to add crufty stuff to this base image if we can help it.)

Reply:
I think either set the path if it's an env var, or if it has to be a file, then you can write a file that just does a `<?php return require("/srv/deskpro/cloud/cloud-config.php");`